website/docs/basics/guides/ckb-tuning-guide.md

```diff
@@ -0,0 +1,99 @@
+---
+id: ckb-tuning-guide
+title: CKB Tuning Guide (For Advanced Users)
+---
+
+**Please note that all features in this guide are advanced user features.
+Some of them are experimental, they may be changed or disappear in the future.
+And please make sure you understand these features clearly before using them.**
+
+## Memory Usage
+
+### Memory Usage during Initial Blockchain Download (aka IBD)
+
+During IBD, CKB will fetch all headers before downloading the blocks.
+
+All headers whose blocks haven't been downloaded are saved in the memory at first, but some of them will cache in the disk if the size of headers reaches a limit.
+
+As an example, we can configure the limit by appending the follow configurations into `ckb.toml` to reduce the memory usage:
+
+```toml
+[network.header_map]
+# The maximum size of data in memory (default: 300_000)
+primary_limit = 3000
+# Disable cache if the size of data in memory less than this threshold (default: 20_000)
+backend_close_threshold = 200
```

Comment:
Please explain what's the unit here. B or MB?

Reply:
Count of `Header`.

p.s. I will update the docs later.

Reply:
ping